From b3391c265ea332c8027997168954ae1c52ffcd37 Mon Sep 17 00:00:00 2001
From: Frederic Boisnard <fredericx.boisnard@intel.com>
Date: Fri, 17 May 2013 17:40:52 +0200
Subject: Allow media process to update AudioComms properties

In order to be able to select a specific XML configuration for the
parameter-framework at runtime, the Android property Audiocomms.PFW.ConfPath
must be updated. However the process where the updated can be achieved
is part of the group media, which is not allowed to update this property.

This patch aims to allow media processes to update Android properties
whose name start with "AudioComms" and "audiocomms".

Issue: 103991
Change-Id: I444e3d510c9c14351b44906e536a3a4eecf540e1
Category: Device Enablement
Domain: AOSP-init
Origin: Internal
Upstream-candidate: No
Signed-off-by: Frederic Boisnard <fredericx.boisnard@intel.com>
---
 init/property_service.c |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/init/property_service.c b/init/property_service.c
index 3c95d4d..61e14c7 100644
--- a/init/property_service.c
+++ b/init/property_service.c
@@ -94,6 +94,8 @@ struct {
     { "persist.security.", AID_SYSTEM,   0 },
     { "persist.service.bdroid.", AID_BLUETOOTH,   0 },
     { "selinux."         , AID_SYSTEM,   0 },
+    { "AudioComms.",       AID_MEDIA,    0 },
+    { "audiocomms.",       AID_MEDIA,    0 },
     { NULL, 0, 0 }
 };
 
-- 
1.7.4.1

